tests/python_client/testcases/test_search.py

```diff
@@ -3637,3 +3637,303 @@ def test_search_string_field_not_primary_is_empty(self, _async):
                                          "ids": insert_ids,
                                          "limit": default_limit,
                                          "_async": _async})
+
+
+class TestsearchPagination(TestcaseBase):
+    """ Test case of search pagination """
+
+    @pytest.fixture(scope="function", params=[0, 10])
+    def offset(self, request):
+        yield request.param
+
+    @pytest.fixture(scope="function", params=[8, 128])
+    def dim(self, request):
+        yield request.param
+
+    @pytest.fixture(scope="function", params=[False, True])
+    def auto_id(self, request):
+        yield request.param
+
+    @pytest.fixture(scope="function", params=[False, True])
+    def _async(self, request):
+        yield request.param
+
+    """
+    ******************************************************************
+    #  The following are valid base cases
+    ******************************************************************
+    """
+
+    @pytest.mark.tags(CaseLabel.L1)
+    @pytest.mark.skip(reason="wait")
```

Comment:
Why skip this case, and what is "wait" meaning?

Reply:
> Why skip this case, and what is "wait" meaning?

There is something wrong with this case and it will not run successfully. I'll look at it later.